modules/apps/object/object-test/src/testFunctional/paths/ObjectModelBuilder.path

```diff
@@ -45,6 +45,31 @@
 		<td>//div[contains(@class,'left-sidebar-current-object-folder-group')]//span[contains(@class,'c-inner') and contains(.,'${key_label}')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>ADD_FIELD_OR_RELATIONSHIP_BUTTON</td>
+		<td>//button[contains(@class, 'dropdown-toggle btn btn-sm btn-secondary') and contains(., 'Add Field or Relationship')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>ADD_FIELD_OR_RELATIONSHIP_BUTTON_OPTION</td>
+		<td>//div[contains(@class, 'dropdown-menu show')]//button[contains(., '${key_optionMenu}')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>LABEL_OBJECT_RELATIONSHIP</td>
```

Comment:
I had to create a more specific macro for this input, because with the existing one the test was writing to the side information bar (which is behind the relationship modal), even though I was using the `SelectFrameTop(); ` function.